.github/workflows/build.yml

```diff
@@ -166,7 +156,7 @@ jobs:
         shell: bash
         run: |
           echo "PYTHON_SUBVERSION=$(echo $PYTHON_VERSION | cut -c 3-)" >> $GITHUB_ENV
-          echo "SINGLE_ACTION_CONFIG=$( [[ $PYTHON_VERSION == 3.8 && $RUNNER_OS == 'macOS' ]] && echo 'True' || echo 'False' )" >> $GITHUB_ENV
+          echo "SINGLE_ACTION_CONFIG=False" >> $GITHUB_ENV
```

Comment:
If this is just always False then we can just remove it, and from line 175 below.